docs/en/stack/data-frames/dataframe-examples.asciidoc

```diff
@@ -0,0 +1,333 @@
+[role="xpack"]
+[testenv="basic"]
+[[dataframe-examples]]
+== {dataframe-cap} examples
+++++
+<titleabbrev>Examples</titleabbrev>
+++++
+
+beta[]
+
+This page provides examples of how to use {dataframe-transforms} to derive useful 
+insights from your data. All the examples use one of the 
+{kibana-ref}/add-sample-data.html[{kib} sample datasets]. For a more detailed, 
+step-by-step example, see 
+<<ecommerce-dataframes,Transforming your data with {dataframes}>>.
+
+* <<example-best-customers>>
+* <<example-airline>>
+* <<example-clientips>>
+
+[float]
+[[example-best-customers]]
+=== Finding your best customers
+
+In this example, we use the eCommerce orders sample dataset to find the customers 
+who spent the most in our hypothetical webshop. Let's transform the data such 
+that the destination index contains the number of orders, the total price of 
```

Comment:
I know I suggested listing these here, but if you find it's too wordy, maybe a solution would be to have a subset, e.g.:

> Let's create/preview?/consider?/look at? a data frame that contains information such as the ..." (and only list a few interesting items).

Reply:
Maybe it is a bit longer this way, but I think it is also more exact. I would leave as is.